.Documentation/articles/innoactive-creator/default-conditions.md

```diff
@@ -23,149 +25,281 @@ See [this article](step-inspector.md) to learn Step Inspector controls.
 
 ### Configuration
 
-- #### Object to collide into
+- #### Object
 
-    This field contains the `Training Scene Object` with the collider inside which you want to move the target object. Make sure you added the `Collider With Trigger Property` component to the game object with the collider. Besides, the collider component needs the `Is Trigger` property to be *enabled*.
+    The `Training Scene Object` to grab. The object needs to have the `Grabbable Property` and a collider component configured. The collider defines the area where the trainee can grab this object.
 
-    [![Object with trigger collider](../images/conditions/object_into_colliders_properties_1.png "")](../images/conditions/object_into_colliders_properties_1.png)
 
-- #### Target object
 
-    This field contains the `Training Scene Object` that you want to move into the *Object to collide into*. Make sure you added the `Training Scene Object` component to the game object in the Unity Inspector. Besides, the object must have a collider component with the `Is Trigger` property *disabled*.
+- #### Collider
 
-    [![Target object with collider](../images/conditions/object_into_colliders_properties_2.png "")](../images/conditions/object_into_colliders_properties_2.png)
+    The `Training Scene Object` with the collider you want to move the `Object` to. Make sure that it has a collider added and that the option `Is Trigger` is enabled.
 
 - #### Required seconds inside
 
-    In this field, you can set the time in seconds the target object should stay within the collider.
-  
+    Set the time in seconds that the `Object` should stay inside the `Collider`.
+
+### Location of this Condition (for Developers)
+
+This behavior is part of the <a href="https://github.com/Innoactive/Basic-Conditions-And-Behaviors" target="_blank">Basic-Conditions-And-Behaviors</a> component. The file is located <a href="https://github.com/Innoactive/Basic-Conditions-And-Behaviors/blob/develop/Runtime/Conditions/PositionalConditions/ObjectInColliderCondition.cs" target="_blank">here</a>.
+
 ------
 
 ## Object Nearby
 
 ### Description
 
-The `Object Nearby` condition triggers when the *tracked object* is within the specified *range* from the *reference object* for the required amount of time while this condition is active.
+This condition is fulfilled when the `Object` is within the specified `Range` of a `Reference object`.
+
+#### Interaction in VR
+
+In order to move the object nearby a collider, trainees need to move their hands/controllers into the object (more precisely its collider) and press the trigger button. While holding the trigger button and moving the arm, the object stays attached to the trainees' hands. Trainees approach with the object the specified collider and release it. The button to trigger can be changed by the template developer as described <a href="https://developers.innoactive.de/documentation/creator/latest/articles/xr/index.html" target="_blank">here</a>.
```

Comment:
I don't understand what the specified collider is :o Maybe I am missing something?